src/visualBuilder/components/FieldToolbar.tsx

```diff
@@ -579,6 +579,9 @@ function FieldToolbarComponent(
                     toolbarRef={toolbarRef}
                     apps={fieldLocationData?.apps || ([] as any[])}
                     position={appListPosition}
+                    domEditStack={fieldLocationData.DomEditStack}
```

Comment:
We should not need the DomEditStack from the builder, we have this data here itself right?